README.md

```diff
@@ -47,9 +47,11 @@ Type: `Element | Element[]`
 The container element, input element or array of elements to get values from.
 
 #### opts
-Type: `Object`
-Default: `{elementSelectors: defaultElementSelectors, shouldStringify: false}`
-An object to override the default settings. `shouldStringify` declares if the returned value should be a string representation of an object or just the object.
+Type: `Object`   
+Default: `{elementSelectors: defaultElementSelectors, shouldStringify: false, shouldReturnAllCheckboxGroupVals: false}`
```

Comment:
[suggestion] For the name of this new option, should it instead be `shouldReturnAllCheckboxVals`. That would be inclusive of single checkboxes that are not part of a checkbox group.

Reply:
What a great suggestion for this exciting new feature!